code/modules/clothing/glasses/_glasses.dm

```diff
@@ -152,6 +152,10 @@
 	if(slot == ITEM_SLOT_EYES)
 		return 1
 
+/obj/item/clothing/glasses/science/suicide_act(mob/living/carbon/user)
+	user.visible_message(span_suicide("[user] is tightening \the [src]'s straps around [user.p_their()] neck! It looks like [user.p_theyre()] trying to commit suicide!"))
+	return OXYLOSS
```

Comment:
?

Reply:
i moved it because the suicide message code was under what it was actually related to instead of under nv goggles

![image](https://user-images.githubusercontent.com/95765134/161384595-053e8a1a-39d1-411c-a314-86296791e538.png)